test/decoders/test_video_decoder_ops.py

```diff
@@ -5,6 +5,7 @@
 # LICENSE file in the root directory of this source tree.
 
 import os
+from functools import partial
```

Comment:
Ditto for the ops tests - although we've been in this situation for a long time because we've had the audio metadata test.